Compile error on macOS

```
src/target/riscv/riscv.c:1171:37: error: length modifier 'L' results in undefined behavior or no effect with
      'x' conversion specifier [-Werror,-Wformat]
        int result = sscanf(CMD_ARGV[0], "%Lx", &address);
                                          ~^~
src/target/riscv/riscv.c:1171:37: note: did you mean to use 'll'?
        int result = sscanf(CMD_ARGV[0], "%Lx", &address);
                                           ^
                                           ll
1 error generated.
```

--- a/src/target/riscv/riscv.c
+++ b/src/target/riscv/riscv.c
@@ -1168,7 +1168,7 @@ COMMAND_HANDLER(riscv_set_scratch_ram)
 	}
 
 	long long unsigned int address;
-	int result = sscanf(CMD_ARGV[0], "%Lx", &address);
+	int result = sscanf(CMD_ARGV[0], "%llx", &address);
 	if (result != (int) strlen(CMD_ARGV[0])) {
 		LOG_ERROR("%s is not a valid address for command.", CMD_ARGV[0]);
 		riscv_use_scratch_ram = false;

